components/admin_console/system_users/system_users.jsx

```diff
@@ -154,32 +141,6 @@ export default class SystemUsers extends React.Component {
         }
     }
 
-    updateUsersFromStore(teamId = this.state.teamId, term = this.state.term) {
```

Comment:
Might be too much to ask but it might worth extending this refactor effort to migrate this component into using redux (and eliminate those use of `...Store`)?

Reply:
I agree that this is desirable, but I'm inclined to consider this out of scope for this PR and the ultimate goal of stopping the config refresh. I think it might be easier to come at this from the angle of eliminating a given store at a time.